std_msgs/Header head

int32 wheel_1_target
int32 wheel_2_target
int32 wheel_3_target
int32 wheel_4_target

int32 wheel_1_rpm
int32 wheel_2_rpm
int32 wheel_3_rpm
int32 wheel_4_rpm